hydromt_wflow/wflow.py

```diff
@@ -3605,7 +3621,7 @@ def read_tables(self, **kwargs):
         if len(fns) > 0:
             for fn in fns:
                 name = basename(fn).split(".")[0]
-                tbl = pd.read_csv(fn)
+                tbl = pd.read_csv(fn, float_precision="round_trip")
```

Comment:
Why this?? And only on the read?
It's not very well documented in pandas but this seems to relate to string precision and not numbers.

Reply:
This has more to do with testing (as tests are written for entire dataframes; with writing them to the drive in between). This causes precision issues. The tables test passed last time out of sheer luck. Setting the precision like this does impact performance (when reading) a little bit, but that is not noticable in this case.